CHANGES.rst

```diff
@@ -8,6 +8,9 @@ Changes
   ``ZopeTransactionEvents``.
   (`#46 <https://github.com/zopefoundation/zope.sqlalchemy/issues/46>`_)
 
+- Reduce ``DeprecationWarning``s with SQLAlchemy 1.4.
```

Comment:
GitHub's ReStructuredText renderer doesn't like this.

restview confirms that this is a problem:

> System Message: WARNING/2 (CHANGES.rst, line 11); backlink
>
> Inline literal start-string without end-string.

```suggestion
- Reduce DeprecationWarnings with SQLAlchemy 1.4.
```